### PR TITLE
Upstream element-animate-list-of-keyframes.html from Blink

### PR DESCRIPTION
**Pull request by alancutter**
Import https://chromium.googlesource.com/chromium/src/+/c0846df098b6a1ba6487d7c351d1f3bb0d6f7690/third_party/WebKit/LayoutTests/web-animations-api/element-animate-list-of-keyframes.html

### CODE REVIEW

File: web-animations/interfaces/KeyframeEffect/processing-a-keyframes-argument.html [hunk 1]
```diff
@@ -175,6 +175,92 @@
...
+    ],
+  },
+];
+
+gEquivalentSyntaxTests.forEach(function({description, indexedKeyframes, sequencedKeyframes}) {
+  test(function(t) {
+    assertEquivalentKeyframeSyntax(indexedKeyframes, sequencedKeyframes);
+  }, 'Equivalent property indexed and sequenced keyframes: ' + description);
+});
+
```
Comment (suzyh) [inline web-animations/interfaces/KeyframeEffect/processing-a-keyframes-argument.html:263]: I think we should also include the test that mixing the two syntaxes throws a TypeError.

Comment (alancutter) [inline web-animations/interfaces/KeyframeEffect/processing-a-keyframes-argument.html:263]: I don't think it should actually throw, instead the array is stringified and used as the value.

Comment (suzyh) [inline web-animations/interfaces/KeyframeEffect/processing-a-keyframes-argument.html:263]: We should verify that with Brian and add a test to illustrate. I'll merge this; maybe raise an issue to make sure we don't forget?

**Comment by suzyh** [comment]
lgtm

File: web-animations/interfaces/KeyframeEffect/processing-a-keyframes-argument.html (hunk 1 above)
Comment (alancutter) [inline web-animations/interfaces/KeyframeEffect/processing-a-keyframes-argument.html:263]: The last test in this file covers that case.

Comment (alancutter) [inline web-animations/interfaces/KeyframeEffect/processing-a-keyframes-argument.html:263]: Sorry, the test at line 168.

Comment (suzyh) [inline web-animations/interfaces/KeyframeEffect/processing-a-keyframes-argument.html:263]: I suppose you're right. Never mind then ;)

**Pull request merged**